halloffame.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+$db = new PDO('mysql:host=db; dbname=halloffame', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+$query = $db->prepare('SELECT `name`, `age`,`nationality`,`goal`,`image` FROM `Players`;');
+$query->execute();
+$fame = $query->fetchAll();
+
+
+?>
+
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="halloffame.css">
+    <link href="https://fonts.googleapis.com/css2?family=Exo+2:ital,wght@0,600;1,700&family=Raleway:wght@100&display=swap" rel="stylesheet">
+    <title>halloffame</title>
+</head>
+<body>
+<div class="navTop">
+    <div class="<img src="halloffame.png"></div>
+<div class="title">
+    <h1> FOOTBALLERS HALL OF FAME </h1>
+</div>
+</div>
+<header>
+    <class="logo"> <img src="new.png" alt= "hall of fame logo"/>
+</header>
+<h3> H.O.F Nominees </h3>
+<body id="container">
+    <div class="Players">
+        <?php foreach ($fame as $player) {
```

Comment:
I'd recommend putting this code into a function inside a separate file and requiring it at the top of this file. 

Additionally, you should build up a string here and return it inside the function rather than echo-ing everything out so you can unit test it!